src/controllers/product.controler.js

```diff
@@ -0,0 +1,49 @@
+
+const productController = (Product) => {
+    // user passed in here would be our user model used to access our database
+
+    const product = (req, res) => {
+        var pdt = new Product();
+        pdt.set("productName", req.body);
+        pdt.set("productPrice", req.body);
+        pdt.set("roductDesc", req.body);
+        pdt.set("productImage", req.body);
+        pdt.set("productCoupon", req.body);
+        pdt.set("productPromo", req.body);
+
+        pdt.save().then((pdt) => {
+            res.status(200).json({ message: "Sucessfully added product : " + pdt });
```

Comment:
also remember to study HTTP status codes and use cases, for insertions, you should return a 201 and not 200